### ISSUE TITLE
False positive for invalid-name

### ISSUE THREAD

**Issue by keiranjprice101** [PyCQA/pylint#3947]
### Steps to reproduce
1. Method named go as below:
```python
    def go(self):
        pass
```
2. This results in `C0103: Method name "go" doesn't conform to snake_case naming style (invalid-name)`

**Comment by dbaty** [comment]
This is a feature, not a bug. :) If you use the [include-naming-hint](http://pylint.pycqa.org/en/latest/user_guide/options.html?highlight=invalid-name#cmdoption-include-naming-hint) option on the command line, pylint will provide a bit more details:

```
C0103: Function name "go" doesn't conform to snake_case naming style ('([^\\W\\dA-Z][^\\WA-Z]{2,}|_[^\\WA-Z]*|__[^\\WA-Z\\d_][^\\WA-Z]+__)$' pattern) (invalid-name)
```

In other words, the name of a function should have at least 3 characters if it does not start with an underscore. You may [customize the regular expressions](https://pylint.pycqa.org/en/latest/user_guide/options.html?highlight=invalid-name#custom-regular-expressions) that are used to check names. Or just disable the warning.

If that's all right, you can close this issue.
